code/game/gamemodes/blob/theblob.dm

```diff
@@ -148,9 +148,7 @@ var/list/blob_looks
 
 /obj/effect/blob/beam_disconnect(var/obj/effect/beam/B)
 	..()
-	apply_beam_damage(B)
```

Comment:
Yeah the problem is its supposed to damage normal things when they move out of the way so they don't out time it though? You could as easily check gcdestroyed/disposed cant you